src/styles/blocks/card.scss

```
@@ -0,0 +1,101 @@
+.card {
+  width: $card-width;
+  border: 1px solid $color-light-grey;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```

```suggestion
  display: flex;
  flex-direction: column;
  align-items: center;
  width: $card-width;
  padding: 32px 16px 16px;
  border: 1px solid $color-light-grey;
  border-radius: 5px;
```

It's a good practice to use Box Model for styling in css...
If you want to read more about it: https://www.w3schools.com/css/css_boxmodel.asp